sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.1.5 (Unreleased)
```

Comment:
version change to 1.2.0

Reply:
also, don't forget to change in _version.py  (assuming the build will fail even if you do :) )

Reply:
good catch, will do!